js/react/components/auth/login/Login.jsx

```diff
@@ -59,7 +59,7 @@ function Login({ modalRef, dispatch }) {
 						<p className="fs-5 mt-1">Join our great movement</p>
 					</div>
 					<button
-						className="w-100 button-modal-outline fs-5 fw-bold"
```

Comment:
Could you leave this one as "button-modal-outline" ? There was a minor design revision. Thanks